action.yml

```diff
@@ -96,4 +97,13 @@ runs:
         working-directory: ${{ inputs.path }}
         if: inputs.install
         run: make install
- 
+
+      # Print errors 
+      - shell: bash
```

Comment:
It would make more sense to me if this block was above the "Install" step.

Reply:
Moved.